managed/services/backup/backup_service.go

```diff
@@ -505,3 +512,61 @@ func (s *Service) prepareBackupJob(
 
 	return res, dbConfig, nil
 }
+
+func (s *Service) checkArtifactModePreconditions(ctx context.Context, artifactID string, pitrTimestamp time.Time) error {
+	artifact, err := models.FindArtifactByID(s.db.Querier, artifactID)
+	if err != nil {
+		return err
+	}
+
+	if artifact.Vendor == string(models.MongoDBServiceType) && artifact.DataModel == models.PhysicalDataModel {
+		return errors.Wrapf(ErrIncompatibleService, "restore of physical backups is not supported for MongoDB yet")
+	}
+	if artifact.Vendor != string(models.MongoDBServiceType) && artifact.Mode == models.PITR {
+		return errors.Wrapf(ErrIncompatibleService, "restore to point in time is only available for MongoDB yet")
```

Comment:
Not sure if something like PITR available for other databases
```suggestion
		return errors.Wrapf(ErrIncompatibleService, "restore to point in time is only available for MongoDB")
```

Reply:
I mean there is o clarity when/if we will support it for something else. Better to not confuse users.